lib/manageiq/appliance_console/database_configuration.rb

```diff
@@ -46,7 +46,9 @@ def run_interactive
       post_activation
       say("\nConfiguration activated successfully.\n")
     rescue RuntimeError => e
-      puts "Configuration failed#{": " + e.message unless e.class == MiqSignalError}"
+      if e.class != MiqNothingChanged
+        puts "Configuration failed#{": " + e.message unless e.class == MiqSignalError}"
+      end
```

Comment:
I don't think we should change this. Just because we didn't change anything doesn't mean that we didn't fail to create the database.

What we're saying here is that we were not able to do what you requested. I think that should imply that we didn't change the state of the system.

Reply:
It's controversial. @yrudman I think both way is fine if we mentioned on the manual, what do you think?

Reply:
yes, changing existing configuration failed, but i think this message message
 `An internal database already exists.
Choose "Reset Internal Database" to reset the existing installation` 
imply that attempt to change was rejected

this message:
`An internal database already exists.
Choose "Reset Internal Database" to reset the existing installation
Configuration failed
`
sounds like we have broke something, and that was complain from user

Reply:
> An internal database already exists. Choose "Reset Internal Database" to reset the existing installation Configuration failed
> sounds like we have broke something, and that was complain from user

I guess I just disagree. It seems to me that this is more of a misunderstanding rather than a flaw in the wording.

Reply:
yes, it is definitely misunderstanding.